src/bundle/Resources/views/themes/admin/ui/component/dropdown/dropdown.html.twig

```diff
@@ -183,6 +183,13 @@
                     {% include list_item_template_path with { choice, custom_form, icon: item_icon, icon_class: item_icon_class } %}
                 {% endfor %}
             </ul>
+
+            <div class="ibexa-dropdown__items-list-filter-empty" hidden>
+                <svg>
```

Comment:
class ibexa-icon?

Reply:
Here there's no point for this class - this class sets width and height of an element to 2rem and I need for this svg to be of dynamic size (based on widget width)